src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsdocumentmanagementapi/config/HmppsDocumentManagementApiExceptionHandler.kt

```diff
@@ -56,8 +57,8 @@ class HmppsDocumentManagementApiExceptionHandler {
   }
 
   @ExceptionHandler(HttpMessageNotReadableException::class)
-  fun handleNoBodyValidationException(e: HttpMessageNotReadableException): ResponseEntity<ErrorResponse> {
-    log.info("Validation exception: Couldn't read request body: {}", e.message)
+  fun handleHttpMessageNotReadableException(e: HttpMessageNotReadableException): ResponseEntity<ErrorResponse> {
+    log.info("HTTP message not readable exception: {}", e.message)
```

Comment:
Out of interesting, why do you not:
`log.error("HTTP message not readable exception", e)`?